tests/clients/test_experiment.py

```diff
@@ -503,6 +504,29 @@ def test_experiment_create_run(mocker):
     )
 
 
+def test_experiment_create_run_experiment_deleted_conflict(mocker):
+    message = "experiment deleted"
+    error_code = "experiment_deleted"
+    response_mock = mocker.Mock()
+    response_mock.json.return_value = {"experimentId": "test-id"}
```

Comment:
Experiment ID is usually an `int` - maybe we should use an int as the mock value here?